src/main/java/io/rpg/model/object/GameObject.java

```diff
@@ -80,6 +82,14 @@ public GameObject(@NotNull String tag, @NotNull Position position, @NotNull Stri
     this.stateChangeObservers = new LinkedHashSet<>();
   }
 
+  public GameObject(@Nullable Position position, @NotNull String tag, @Nullable String assetPath, int strength) {
+    this.position = position;
+    this.tag = tag;
+    this.assetPath = assetPath;
+    this.strength = strength;
+    this.stateChangeObservers = new LinkedHashSet<>();
+  }
```

Comment:
I believe this is only temporary solution. We don't want constructor of this form in final version. Maybe add some comments.
```suggestion
    // TODO: remove this ctor when generic properties are implemented
  public GameObject(@Nullable Position position, @NotNull String tag, @Nullable String assetPath, int strength) {
    this.position = position;
    this.tag = tag;
    this.assetPath = assetPath;
    this.strength = strength;
    this.stateChangeObservers = new LinkedHashSet<>();
  }
```